contentcuration/contentcuration/viewsets/contentnode.py

```diff
@@ -254,15 +254,15 @@ def update(self, queryset, all_validated_data):
         return all_objects
 
 
-class ThresholdField(CharField):
+class ThresholdField(Field):
```

Comment:
What spurred changes to this class? Are they still required?

Reply:
I was getting "errors":{"extra_fields":{"options":{"completion_criteria":{"threshold":["Not a valid string."]}}}}}]}' when I was running the tests. It seems like we need to add special handling in the to_internal_value. This was the work around suggested by @rtibbles sir.